util/network-devp2p/src/disk.rs

```diff
@@ -0,0 +1,94 @@
+use log::*;
+use parity_crypto::publickey::Secret;
+use parity_path::restrict_permissions_owner;
+use std::fmt::Debug;
+use std::fs;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+use std::str::FromStr;
+
+pub trait DiskEntity: FromStr {
+	const PATH: &'static str;
+	const DESC: &'static str;
+
+	fn to_repr(&self) -> String;
+}
+
+impl DiskEntity for Secret {
+	const PATH: &'static str = "key";
```

Comment:
"key" strikes me as a bit too generic. Can we use something more self-explaining?

Reply:
This would break backwards compatibility.

Reply:
got it, too bad.